source/element-parameters.md

```diff
@@ -66,387 +69,68 @@ Element:
 ```
 
 %---------------------------------------------------------------------------------------------------
-(s:ac.kicker.params)=
-## ACKickerP:  AC-Kicker Parameters
+## Dependent and User Set parameters.
 
-In Construction...
+Some parameters are set in the lattice file. These parameters are called "User settable". 
+Some parameters are not settable and will be computed by lattice expansion code. 
```

Comment:
```suggestion
Some parameters cannot be set by the user and will be computed by lattice expansion code. 
```

Reply:
changed to:
```
Some parameters will be computed by the Translator during lattice expansion. Such parameters
may not be set by the user.
```

Reply:
This brings up the question whether it is an error for Translator computed quantities to be present in the lattice file. I can see a use case where the User wants to print out the Translated file complete with all computed quantities.